readthedocs/projects/constants.py

```diff
@@ -250,12 +250,12 @@
     ('zh', 'Chinese'),
     ('zu', 'Zulu'),
     # Try these to test our non-2 letter language support
-    ('nb_NO', 'Norwegian Bokmal'),
-    ('pt_BR', 'Brazilian Portuguese'),
-    ('es_MX', 'Mexican Spanish'),
-    ('uk_UA', 'Ukrainian'),
-    ('zh_CN', 'Simplified Chinese'),
-    ('zh_TW', 'Traditional Chinese'),
+    ('nb-NO', 'Norwegian Bokmal'),
+    ('pt-BR', 'Brazilian Portuguese'),
+    ('es-MX', 'Mexican Spanish'),
+    ('uk-UA', 'Ukrainian'),
+    ('zh-CN', 'Simplified Chinese'),
+    ('zh-TW', 'Traditional Chinese'),
```

Comment:
I don't think we can change this.

`LANGUAGES` is used for URLs, and by changing this constant we will be breaking many URLs. Also, these constant matches the `language=` setting from Sphinx.

Reply:
@humitos we can add a method to `sitemap_xml` view to change the lang if its not in the correct format. somthing like `if lang == 'zh_CN': return 'zh-CN'`. It would be a hacky process though. What do you think we can do here?

Reply:
Sounds hacky, but good enough to me considering that it's what we need and it's encapsulated inside the `sitemap_xml`.

Is the only thing that needs to be replaced `_` by `-`? Did you check the sitemap reference?

What about the rest of the values? are they valid?

Reply:
@humitos  yes, we only need to replaced `_` by `-` . Updated the PR please have a look.